### PR TITLE
Deployment files

### PR DESCRIPTION
**Pull request by anumehaagrawal**
### Description
In this PR I have added files for deployment

### Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.

### Checklist:
**Delete irrelevant options.**

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by anumehaagrawal** [comment]
@Tharangi @Rupinderwadali  @kanmodi  Can you please review my PR ?

File: main.html [hunk 1]
```diff
@@ -24,23 +24,27 @@ <h1><center>GHC 2018</center></h1>
     </div>
     
     <form method="post" action="/addname">
-      <label>Enter Your email ID</label><br>
       <input type="text" name="name" placeholder="Enter email ID" required style="color: white;">
+      <div style="margin: 15px;"></div>
       <div class="g-recaptcha" data-sitekey="6LcE6mIUAAAAAJtNhVBc-2AIWl69s6GZ383ZBx2r" id="rcaptcha" data-callback="enableBtn"></div>
       <span id="captcha" style="color:red" /></span>
```
Comment (Rupinderwadali) [inline main.html]: I think there is an unwanted "/" here. Remove it.

**Comment by anumehaagrawal** [comment]
@Tharangi @Rupinderwadali @kanmodi  Can we merge this PR ?

**Comment by kanmodi** [comment]
Merged!

**Pull request merged**